cdc/api/v2/api_helpers.go

```diff
@@ -551,5 +579,9 @@ func (h APIV2HelpersImpl) getVerifiedTables(
 		return nil, nil, err
 	}
 
+	if ctx.Err() != nil {
```

Comment:
Perhaps the redundant code can be reduced through a common `ctxChecker` function? eg:
```go
func ctxChecker(ctx, fn func() error) {
    if fn() != nil {
		return nil, errors.Trace(err)
	}
	if ctx.Err() != nil {
		return nil, errors.Trace(ctx.Err())
	}
}
```